repository.h

```diff
@@ -41,6 +41,8 @@ struct repo_settings {
 	enum fetch_negotiation_setting fetch_negotiation_algorithm;
 
 	int core_multi_pack_index;
+
+	int use_internal_fsmonitor;
```

Comment:
Nit.   Would this be better as `use_fsmonitor_feature` or `use_fsmonitor_daemon` ??

Reply:
I guess I'm thinking that we got rid of the term `:internal:` from the config setting, so it doesn't have as much meaning now.

Reply:
I renamed it to `use_builtin_fsmonitor` after some face-to-face discussion.